app/helpers/api/ordapi/postOrderItemBulk.test.js

```diff
@@ -77,9 +77,9 @@ describe('postOrderItemBulk', () => {
         formData,
       });
 
-      expect(postData.mock.calls.length).toEqual(1);
-      expect(postData).toHaveBeenCalledWith({
-        endpoint: `${orderApiUrl}/api/v1/orders/order1/order-items/batch`,
+      expect(putData.mock.calls.length).toEqual(1);
+      expect(putData).toHaveBeenCalledWith({
+        endpoint: `${orderApiUrl}/api/v1/orders/order1/order-items/item-1`,
         body: [{
           ...selectedPrice,
           serviceRecipient,
```

Comment:
The new contract no longer expects an array so it looks like further changes are required.

The request body (in pseudo-form) should be like this:


```json
{
    "catalogueItemName": "value",
    "serviceRecipients": [
        {
            "odsCode": "ABC1",
            "quantity": 1,
            "otherProperties..."
        }
    ]
    "otherProperties..."
}
```

The updated contract is now on the [wiki](https://dev.azure.com/BuyingCatalog/Buying%20Catalogue/_wiki/wikis/Wiki/183/BE-ORDAPI-bulk?anchor=put-%2Fapi%2Fv1%2Forders%2F%7Bcalloffid%7D%2Forder-items%2F%7Bcatalogueitemid%7D).